tests/fixtures/optimizer/annotate_functions.sql

```diff
@@ -1555,10 +1555,34 @@ TEXT;
 REVERSE(tbl.bin_col);
 BINARY;
 
+# dialect: snowflake
+REVERSE(NULL);
+NULL;
```

Comment:
I think this was added by mistake– I got rid of it in main. Let's git rebase off of it.